pkg/operatorstatus/operatorstatus.go

```diff
@@ -103,6 +103,11 @@ func (o *ClusterOperatorHandler) setOperatorStatus(condtype configv1.ClusterStat
 			state = &configv1.ClusterOperator{}
 			state.Name = ClusterOperatorName
 
+			state.Status.RelatedObjects = []configv1.ObjectReference{
+				{Group: v1.GroupName, Resource: "configs", Name: "cluster"},
+				{Resource: "namespaces", Name: "openshift-cluster-samples-operator"},
```

Comment:
you probably want to gather the openshift namespace too.

Reply:
good call

Reply:
update pushed @bparees